t/gene_tree.t

```diff
@@ -338,22 +326,6 @@ sub find_leaf {
     }
 }
 
-my $json1 = json_GET(
-    '/genetree/member/id/ENSG00000176515?compara=homology&clusterset_id=default',
-    'Gene-tree (ncRNA) by ID. Explicitly require the default clusterset_id',
-);
-is(count_leaves($json1->{tree}), 69, 'Got all the leaves');
-my ($human_leaf) = find_leaf($json1->{'tree'}, 'ENSG00000176515');
```

Comment:
If `my $human_leaf` is used by any of the remaining unit tests, it may need to be declared where it is first used.